internal/generator/fluentd/output/cloudwatch/cloudwatch.go

```diff
@@ -150,3 +162,64 @@ func LogGroupName(o logging.OutputSpec) string {
 	}
 	return ""
 }
+
+func HasAwsCredentialsKey(secret *corev1.Secret) bool {
+	return security.HasKeys(secret, constants.AWSCredentialsKey)
+}
+
+// ParseRoleArn search for 'arn:aws:iam:: in the credentials string and return string up to end of line
+func ParseRoleArn(secret *corev1.Secret) string {
```

Comment:
Consider adding an isolated unit test for this helper function

Reply:
added cloudwatch_secret_test.go